app/views/content/non-uk-teachers/tips-on-applying-for-teacher-training-as-a-non-uk-citizen.md

```diff
@@ -0,0 +1,205 @@
+
+## Finding the right course for you  
+
+[Find postgraduate teacher training courses](https://www.gov.uk/find-postgraduate-teacher-training-courses) lists courses leading to [English qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts). 
+
+If you need a visa, remember to filter your search results by visa sponsorship. 
+
+You can also search courses by: 
+
+* location 
+
+* teaching subject 
+
+* age group 
+
+* courses which pay a salary 
+
+## How to increase your chance of success
+
+### Do your research before you apply 
+
+You can read up on: 
+
+* [teacher training application](/how-to-apply-for-teacher-training/teacher-training-application) 
+
+* [teacher training personal statement](/how-to-apply-for-teacher-training/teacher-training-personal-statement) 
+
+* [teacher training references](/how-to-apply-for-teacher-training/teacher-training-references) 
+
+* [when to apply for teacher training](/how-to-apply-for-teacher-training/when-to-apply-for-teacher-training) 
+
+* [teacher training interviews](/how-to-apply-for-teacher-training/teacher-training-interview) 
+
+* [subject knowledge enhancement](/how-to-apply-for-teacher-training/subject-knowledge-enhancement)
+
+* [what to do if your application is unsuccessful](/how-to-apply-for-teacher-training/if-your-application-is-unsuccessful) 
+
+ 
+Email your chosen provider to check they can sponsor you, then apply for your teacher training course. 
+
+
+### Check your qualifications 
+
+Make sure you have the right qualifications and apply for a [statement of comparability](/non-uk-teachers/non-uk-qualifications) proving your school and university qualifications are the same standard as UK GCSEs and a UK bachelor’s degree.
+
+
+### Research English education 
+
+Even if you’ve never spent time in an English school, teacher training providers will want to know you’re familiar with: 
+
+* [types of English school](https://www.gov.uk/types-of-school) 
+
+* [the English curriculum](https://www.gov.uk/national-curriculum)
+
+* [English qualification levels](https://www.gov.uk/what-different-qualification-levels-mean/list-of-qualification-levels)
+
+* [behaviour and discipline in schools](https://www.gov.uk/government/publications/behaviour-in-schools--2) 
+
+* [Office for Standards in Education, Children’s Services and Skills (Ofsted)](https://www.gov.uk/government/organisations/ofsted?)
+
+You can learn more by: 
+
+* visiting [The Education Endowment Foundation](https://educationendowmentfoundation.org.uk/guidance-for-teachers) – a charity gathering international evidence on teaching 5 to 16 year olds 
+
+* visiting [Times Educational Supplement (TES)](https://www.tes.com/en-gb) – information and resources for teachers 
+
+* [joining the Aspiring Teachers Forum on Facebook](https://www.facebook.com/groups/1357146377672255/)
+
+* [reading interviews with real teachers](https://getintoteaching.education.gov.uk/blog) 
```

Comment:
@laurahermionetennant92 Internal links shouldn't have the full domain - this is causing a build error